dumb.scss

```diff
@@ -0,0 +1 @@
+.dumb { .scss { .is { .dumb {  font-size: 700em; } } } }
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Selector should have depth of applicability no greater than 2, but was 4